easybuild/easyconfigs/p/Python/Python-2.7.12-foss-2017a.eb

```diff
@@ -0,0 +1,293 @@
+# Built with EasyBuild version 3.1.1 on 2017-04-21_17-31-01
+# Built with EasyBuild version 3.1.1 on 2017-04-20_11-17-51
+# Built with EasyBuild version 3.1.1 on 2017-04-03_09-46-50
+name = 'Python'
+version = '2.7.12'
+
+homepage = 'http://python.org/'
+description = """Python is a programming language that lets you work more quickly and integrate your systems
+ more effectively."""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+
+# python needs bzip2 to build the bz2 package
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.11'),
+    ('libreadline', '6.3'),
+    ('ncurses', '6.0'),
+    ('SQLite', '3.13.0'),
+    ('Tk', '8.6.5'),  # this requires a full X11 stack
```

Comment:
@henkela The easyconfig for this Tk version includes a dependency on zlib 1.2.8, which conflicts with the zlib used in this easyconfig file. 

Besides that, any reason you are not porting the easyconfig for Python 2.7.13 we have with the `intel/2017a` toolchain to `foss/2017a` instead?

We took a slightly different approach there, by avoiding to include Tk as a dependency since that requires X11 (which is kind of silly as a dep for Python). That does mean the `Tkinter` part of Python is not available (see also  #4225).